pkg/hooks/hooks_test.go

```diff
@@ -97,11 +97,11 @@ func testManifest() manifest.Manifest {
 }
 
 func TestHookRunWithTimeout(t *testing.T) {
-	timeout := 1 * time.Millisecond
-	sleep := "1" // 1 second sleep to be executed by the script
+	timeout := 1 * time.Second
+	sleep := "5" // 1 second sleep to be executed by the script
```

Comment:
nit: the comment still says 1 second

Reply:
Thanks, I fixed the comment to reflect why the number is larger rather than the timeout value